<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow7</class>
 <widget class="QMainWindow" name="MainWindow7">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>L-System Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="loadFileButton">
      <property name="text">
       <string>Загрузить L-систему</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="iterationSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="drawArea">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
